lib/Configuration.php

```diff
@@ -253,14 +250,15 @@ public function setIgnoreInvalidReferences(bool $ignoreInvalidReferences): void
         $this->ignoreInvalidReferences = $ignoreInvalidReferences;
     }
 
+    /** @deprecated The indenter is not maintained anymore and was removed. Indenting is not supported anymore */
     public function setIndentHTML(bool $indentHTML): void
```

Comment:
It's OK to remove this entirely IMO: 0.6.0 is a new major release (that's how semver works when the version number starts with a zero)